envConfig/live.env

```diff
@@ -3,6 +3,7 @@ SIMORGH_ASSETS_MANIFEST_PATH=build/assets-live.json
 SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=https://news.files.bbci.co.uk
 SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/include/articles/public
 SIMORGH_ATI_BASE_URL=https://a1.api.bbc.co.uk/hit.xiti?
+SIMORGH_MPULSE_API_KEY=JLSEX-R76DK-JXL3A-UB7YH-NPSEW
```

Comment:
Emm, shouldn't this be a secret?

Reply:
Nope. The API key is public. Akamai limit by domain on their end.

Reply:
It's accessible on the client by necessity so cannot really be a secret.

Reply:
Alright dab it is